collateral-info.yaml

```diff
@@ -99,6 +99,13 @@ collateral-defs:
     collateral-shortname: Wrapped Lido Staked Ether
     collateral-symbol: WSTETH
     collateral-info-url: https://lido.fi/ethereum
+    
+  RETH: &RETH
+    <<: *ONCHAIN
```

Comment:
Feels scummy giving RETH *CUSTODIAL-ONCHAIN, but I don't want to spend time adding more subtypes right now, so just gave it parent type.